hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metadata/HoodieBackedTableMetadataWriter.java

```diff
@@ -734,6 +736,7 @@ private void initializeFileGroups(HoodieTableMetaClient dataMetaClient, Metadata
       }
     } catch (FileNotFoundException e) {
       // If the partition did not exist yet, it will be created below
+      LOG.warn("Exception seen while removing existing file groups in partition {} ", partitionPath.getName(), e);
```

Comment:
this catches FileNotFoundException ....  so I feel this log is not necessary.

Reply:
removed